assets/prototype/application/ui/components/display/entityList/filterBar/Collapsible.js

```diff
@@ -9,7 +9,7 @@ import { __ } from '@eventespresso/i18n';
 /**
  * Internal dependencies
  */
-import { useRect } from '@eventespresso/hooks';
+import useRect from '../../../../../hooks/useRect';
```

Comment:
Fix for: 
![14_jan_5](https://user-images.githubusercontent.com/1477453/72343450-92623680-36d7-11ea-92ab-429b77e727b0.png)